### ISSUE TITLE
Differential Cyanosis

### ISSUE THREAD

**Issue by cmrn-rhi** [obophenotype/human-phenotype-ontology#7120]
**Preferred term label:**
differential cyanosis

**Synonyms**
asymmetrical cyanosis

**Definition (free text, please give PubMed ID)**
Asymmetrical bluish discoloration between the upper and lower extremities. [PMID: 32644593]

**Parent term (use hpo.jax.org/app)**
HP:0000961 - Cyanosis

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-9578-0788

**Comment by pnrobinson** [comment]
adding with definition
Differential degree of bluish discoloration between the upper and lower extremities.

(symmetry is left right -- there is no symmetry between arms and legs)
